graphql-server/src/schemas/custom.ts

```diff
@@ -636,6 +636,7 @@ const editableSampleMetadataFields = new Set([
   "collectionYear",
   "sampleOrigin",
   "tissueLocation",
+  "sex",
```

Comment:
This fixes an unrelated bug that prevents users from editing the `sex` field of SampleMetadata.